### PR TITLE
[Backport] Backport #45 to 2.x

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
### Description
Backport #15 and #45 to 2.x
 
### Issues Resolved
N/A
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -119,47 +116,61 @@ publishing {
...
-                  name = "The Apache License, Version 2.0"
-                  url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                name = "OpenSearch SQL JDBC Driver"
+                packaging = "jar"
+                url = "https://github.com/opensearch-project/sql/sql-jdbc"
+                description = "OpenSearch SQL JDBC driver"
+                scm {
+                    connection = "scm:git@github.com:opensearch-project/sql.git"
+                    developerConnection = "scm:git@github.com:opensearch-project/sql.git"
+                    url = "git@github.com:opensearch-project/sql.git"
```
Comment (YANG-DB) [inline build.gradle:126]: this url should be /sql-jdbc

File: build.gradle [hunk 2]
```diff
@@ -119,47 +116,61 @@ publishing {
...
-              }
-              developers {
-                developer {
-                  id = "amazonwebservices"
-                  organization = "Amazon Web Services"
-                  organizationUrl = "https://aws.amazon.com"
+                developers {
+                    developer {
+                        name = 'OpenSearch'
+                        url = 'https://github.com/opensearch-project/sql'
```
Comment (YANG-DB) [inline build.gradle:137]: same here https://github.com/opensearch-project/sql-jdbc

**Review by YANG-DB — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (Yury-Fridlyand) [reply build.gradle:126]: I don't change repo link here. I have another upcoming ticket to fix that.

File: build.gradle (hunk 2 above)
Comment (Yury-Fridlyand) [reply build.gradle:137]: I don't change repo link here. I have another upcoming ticket to fix that.

File: build.gradle (hunk 1 above)
Comment (Yury-Fridlyand) [reply build.gradle:126]: Link is fixed in https://github.com/opensearch-project/sql-jdbc/pull/56

File: build.gradle (hunk 2 above)
Comment (Yury-Fridlyand) [reply build.gradle:137]: Link is fixed in https://github.com/opensearch-project/sql-jdbc/pull/56

**Review by MaxKsyunz — APPROVED** [review]
Any reason this can't be merged?

The change to repository url is already merged (PR #56)

**Comment by dai-chen** [comment]
wondering is backport and 2.x branch still relevant after we split repo?

**Comment by Yury-Fridlyand** [comment]
Yes if we are going to use 2.x branch here.

**Comment by Yury-Fridlyand** [comment]
No need in 2.x branch.

**Pull request closed without merging**